### ISSUE TITLE
Access to /sys/kernel/security/ is only effective for the current session

### ISSUE THREAD

**Issue by utkarsh2102** [keylime/keylime-deb#9]
Hello,

[CC: @ibmcb]

I see there's:
https://github.com/keylime/keylime-deb/blob/219cad99509c0c2c9dabae61cbfd2e64b34754da/postinst#L33-L35

But this is only in effect for the current session. As soon as there's a reboot, these changes are essentially lost. So I believe it'd be a good idea to set this via the systemd services as well?

Something along the lines of:
`ExecStartPre=-/bin/chown -R tss:tss /sys/kernel/security/tpm0 `

What do you think?

**Comment by galmasi** [comment]
I agree. We have a particular blind spot in this regard, because our deployments are net-booted, and setting something for a session is the same as setting it forever. Please feel free to submit a PR with the change.

**Comment by utkarsh2102** [comment]
Hi,

Thanks. Actually it's best if this is implemented via a udev rule. I'll work on it and let you know when it's ready.
Though I wanted to ask if `/sys/kernel/security/tpm0 ` and `/sys/kernel/security/ima ` is being used somewhere? (And how?)
I couldn't understand its need on a quick look; sorry if it was straightforward and if I am missing something.

**Comment by galmasi** [comment]
We also tried a udev rule and it proved complicated, but you know best. 

We (keylime) are the exclusive users of /sys/kernel/security/ima and tpm0/binary_bios_measurements.
